src/lib.rs

```diff
@@ -1421,7 +1421,7 @@ impl GooseAttack {
                         self.host.clone(),
                     )?;
                     let user = GooseUser::single(base_url, &self.configuration)?;
-                    let function = t.function;
+                    let function = &t.function;
```

Comment:
We don’t need to clone that because rust can statically determine that this can never be accessed in parallel and automatically derefs it **pretty slick**.

Reply:
rust++ :smile: